spec/spec_helper.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
+  end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+
+  config.shared_context_metadata_behavior = :apply_to_host_groups
```

Comment:
I'd recommend configuring the DatabaseCleaner gem here to clear out the database. It's recommended to do this once before the suite starts running: 
```
  config.before :suite do
    DatabaseCleaner.strategy = :transaction
    DatabaseCleaner.clean_with(:truncation)
  end
```
And once before each example runs:
```
before(:each) do
  DatabaseCleaner.start
end

after(:each) do
  DatabaseCleaner.clean
end
```

I liked the info available here on Database cleaning strategies: http://stackoverflow.com/questions/10904996/difference-between-truncation-transaction-and-deletion-database-strategies